base_station/gui/src/components/AutonJoystickReading.vue

```diff
@@ -0,0 +1,44 @@
+<template>
```

Comment:
We should try to avoid having these commits in your PR since they are not your commits. This goes for a number of the files here.

Reply:
I think these stray commits are because his branch is out of date with respect to `umrover:master`. A `git pull --rebase upstream master` should fix these issues, when i did it when testing his code with my travis updates those extra commits disappeared.